### PR TITLE
Update dependencies and regenerate package-lock.json

### PR DESCRIPTION
**Pull request by XhmikosR**
* @fortawesome/fontawesome-free 5.13.0
* @types/webpack 4.41.11
* preact 10.4.0
* stylelint-scss 3.17.0

This fixes all npm security vulnerabilities and flattens the deps tree (-133 packages)

@squidfunk unfortunately, you will hit the unflattened deps situation again in the future. I also hit it where dependabot is used. Not a huge deal, but after some time you might need to do this yourself again.

### CODE REVIEW

File: material/base.html [hunk 1]
```diff
@@ -178,8 +178,8 @@ <h1>{{ page.title | default(config.site_name, true)}}</h1>
       {% endblock %}
     </div>
     {% block scripts %}
-      <script src="{{ 'assets/javascripts/vendor.606193c2.min.js' | url }}"></script>
-      <script src="{{ 'assets/javascripts/bundle.7af7e239.min.js' | url }}"></script>
+      <script src="{{ 'assets/javascripts/vendor.js' | url }}"></script>
```
Comment (squidfunk) [inline material/base.html:181]: Somehow, the versioning of the vendor bundle got lost. Also from `manifest.json`.

Comment (XhmikosR) [reply material/base.html:181]: Hmm, I just ran `npm run build` and this is what I got. I also get the same on master. I will revert the `npm run build` part for now. It probably is related to Windows. I'll see if I can pinpoint the cause later.

**Comment by squidfunk** [comment]
This branch has conflicts, I'm unable to merge.

**Comment by XhmikosR** [comment]
It seems you pushed it manually to master.

**Comment by squidfunk** [comment]
Weird. I merged via the web interface and then GitHub told me: error merging. I didn't push anything.

**Comment by XhmikosR** [comment]
Maybe it was a hiccup, because I do see the PR merged https://github.com/squidfunk/mkdocs-material/commits/master

**Pull request closed without merging**